### DISCUSSION TITLE
Need some help in postgresql

### DISCUSSION THREAD

**Discussion by maskmanlucifer** [EddieHubCommunity/support#2266] [Q&A]
Hey guys i started working with postgresql recently.
I did installed postgresql on my laptop and created some database and tables.
But i am facing some problem when i am trying to connect postgresql with nodejs.
Error i am facing :  ECONNREFUSED 127.0.0.1:5412
```
const pool = new Pool({
  user:'postgres',
  password:'xxxx',
  database:"perntodo",
  host:"127.0.0.1" // tried localhost and 127.0.0.1 both,
  port:5412
});

```
OS -- Linux Mint 
I am running my nodejs server on localhost at 5000 port.

**Comment by eddiejaoude** [comment]
I think the Postgres port is `5432`
